test/actions/metaplex.test.ts

```diff
@@ -0,0 +1,129 @@
+import { jest } from '@jest/globals';
+import { TupleNumericType } from '@metaplex/utils';
+import { AccountLayout, NATIVE_MINT } from '@solana/spl-token';
+import { Keypair, sendAndConfirmTransaction } from '@solana/web3.js';
+import BN from 'bn.js';
+import { Connection } from '../../src';
+import {
+  Store,
+  SetStore,
+  WhitelistedCreator,
+  SetWhitelistedCreator,
+  AuctionManager,
+  AuctionWinnerTokenTypeTracker,
+  InitAuctionManagerV2,
+  StartAuction,
+} from '../../src/programs/metaplex';
+import { Auction } from '../../src/programs/auction';
+import { CreateTokenAccount, Transaction } from '../../src/programs/shared';
+import { FEE_PAYER, VAULT_PUBKEY } from '../utils';
+
+describe.skip('Metaplex transactions', () => {
```

Comment:
Why skip?

Reply:
I just moved this file.
I assume it was commented due to instability. Since there're requests to the network.